.gitmodules

```diff
@@ -2,5 +2,7 @@
   path = spack
   #url = https://github.com/spack/spack
   #branch = develop
-  url = https://github.com/NOAA-EMC/spack.git
-  branch = jcsda_emc_spack_stack
+  #url = https://github.com/NOAA-EMC/spack.git
+  #branch = jcsda_emc_spack_stack
+  url = https://github.com/climbfuji/spack
```

Comment:
Do we want to change the submodule here?

Reply:
Temporarily, I need to create a PR for our internal spack repo. I am cherry-picking the commits that I submit to the official spack repo in parallel to avoid merge conflicts down the road.